beam_sink/sinks/mysql.py

```diff
@@ -96,6 +116,9 @@ def finish_bundle(self):
 
 
 class _Insert(_PutFn):
+    """Insert.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line](https://app.codacy.com/manual/lislemitchell/beam-sink/pullRequest?prid=5069736)